src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-domain-setting-modal/smb-domain-setting-modal.component.html

```diff
@@ -11,7 +10,8 @@
           #formDir="ngForm"
           [formGroup]="domainSettingsForm"
           novalidate>
-      <div cdsModalContent>
+      <div cdsModalContent
+           class="scrollable">
```

Comment:
I've seen this issue before. What I remember doing is to move the `<form` inside the `<div cdsModalContent> and that sort of fixed the issue in the form. Here is [the code](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-form/cephfs-subvolume-form.component.html#L11). It looks like a carbon issue.


One thing that worries me in the code I shared is the validation. because the form submission is outside the form.. But if we were to go with your implementation, I'd be in favor of generalizing it and maybe even override the carbon class associated with the modal to make it scroll after a particular point, wdyt?

Reply:
You are right, that did solve the issue, thanks!

Reply:
I think then we dont need to remove the [hasScrollingContent]?

Reply:
Yup, you are right, added it back